geo/Geoserver.py

```diff
@@ -75,10 +75,31 @@ def get_coveragestore(self, coveragestore_name, workspace):
         except Exception as e:
             return 'Error: {}'.format(e)
 
+    def get_name_workspace(self,workspace):
```

Comment:
Please change the function name to `get_workspace`.

Reply:
Ready!